libparsec/crates/platform_device_loader/src/web/wrapper.rs

```diff
@@ -0,0 +1,432 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use std::{
+    ffi::OsStr,
+    ops::Deref,
+    path::{Path, PathBuf},
+};
+
+use libparsec_platform_async::{
+    future::{self, FutureExt},
+    stream::{Stream, StreamExt},
+};
+use wasm_bindgen_futures::{stream::JsStream, JsFuture};
+use web_sys::{js_sys, wasm_bindgen::JsCast, DomException};
+
+use super::error::{
+    CastError, GetDirectoryHandleError, GetFileHandleError, GetRootDirectoryError, ReadToEndError,
+    RemoveEntryError, WriteAllError,
+};
+
+#[derive(Clone)]
+pub struct Directory {
```

Comment:
I guess all the wrapper stuff are internal helpers only needed by the web implementation, so there visibility should be `pub(super)` (or at least `pub(crate)`)

Reply:
The module itself (wrapper) is not exposed so it's good ?